src/app/file-upload/file-upload.component.html

```diff
@@ -2,7 +2,7 @@
   <ng-container *ngIf="maxFiles > 0">Maximum of {{maxFiles}} files. </ng-container>
   <ng-container *ngIf="maxSize > 0">File size up to {{maxSize}}MB each.</ng-container>
   <br>
-  <ng-container *ngIf="fileExt.length > 0">Acceptable file types: {{fileExt.slice().join(', ')}}</ng-container>
+  <!-- <ng-container *ngIf="fileExt.length > 0">Acceptable file types: {{fileExt.splice().join(', ')}}</ng-container> -->
```

Comment:
Commented out? Is this meant for future enhancements?

Reply:
I was getting a lot of errors with this when using it in the context of file upload modal. I'm investigating for now and will uncomment later. Thanks for the reminder.